src/com/aca/ArmineArzumanyan/homeWork_1/repository/Repository.java

```diff
@@ -0,0 +1,16 @@
+package com.aca.ArmineArzumanyan.homeWork_1.repository;
+
+import java.util.List;
+
+public interface Repository<T> {
+
+    void printItemList(List<T> itemList);
```

Comment:
Being honest I don't see any hard reason to separate this interface which name doesn't hint me anything about its meaning